README.md

```diff
@@ -62,6 +62,8 @@ What about a whole **directory**? `What` can handle that too! It will **recursiv
 
 **Exporting** You can export to json using `what --json` and results can be sent directly to a file using `what --json > file.json`.
 
+**Boundaryless mode** `What` has a special mode to match identifiable information within characters. By default, it is enabled in CLI but disabled in API. Use `what --help` or refer to [API Documentation](https://github.com/bee-san/pyWhat/wiki/API) for more information.
```

Comment:
I don't think that the **Anything** section is large enough to move it to the wiki. On the other hand, it may be simplified to just a list of all the features with zero explanation and a note about what users need to do to get CLI or API info. Your thoughts, @bee-san?